risc0/groth16/src/verifier.rs

```diff
@@ -158,7 +158,14 @@ impl Verifier {
 
 /// Verifying key for Groth16 proofs.
 #[derive(Clone, Debug, Serialize, Deserialize)]
-pub struct Fr(#[serde(with = "serde_ark")] pub(crate) ark_bn254::Fr);
+pub struct Fr(#[serde(with = "serde_ark")] pub ark_bn254::Fr);
```

Comment:
```suggestion
pub struct Fr(#[serde(with = "serde_ark")] ark_bn254::Fr);
```

This needs to remain private since `ark_fr` provides access in an unstable way.